tests/cli-self-update.rs

```diff
@@ -38,8 +37,8 @@ pub fn setup(f: &Fn(&Config)) {
 
         // An windows these tests mess with the user's PATH. Save
         // and restore them here to keep from trashing things.
-        let ref saved_path = get_path();
```

Comment:
I believe I also converted it to a value binding and still got the crash, but can't be sure.